news/5853.hooks.rst

```diff
@@ -0,0 +1,8 @@
+* Add ``PostGraphAPI.analysis`` attribute.
+  Hooks can access the ``Analysis`` object
+  through the ``hook()`` function.
+* Hooks can access a ``Analysis.hooksconfig`` attribute
+  assigned on ``Analysis`` construction.
+  
+  A helper function ``get_hook_config()``
+  was defined in ``utils.hooks`` to get the config.
```

Comment:
We'll need some docs for this too. Can you add a sentence to [this paragraph](https://github.com/pyinstaller/pyinstaller/blob/develop/doc/hooks.rst#the-hookhook_api-function) to saying something like:
```rst
Hooks may access the user parameters, given in the ``hooksconfig`` argument in
the spec file, by calling :func:`get_hook_config()` inside a `hook()` function.

.. autofunction:: PyInstaller.utils.hooks.get_hook_config

```

Then you should be able to use fancy cross references like ``:func:`~PyInstaller.utils.hooks.get_hook_config` `` in this news entry.